tests/tests_integration/test_api/test_diagrams.py

```diff
@@ -9,8 +9,11 @@
     DiagramDetectResults,
     FileReference,
 )
+from cognite.client.data_classes.data_modeling import NodeId
 
 PNID_FILE_ID = 3261066797848581
+DIAGRAM_SPACE = "diagram_space"
```

Comment:
Can you make a fixture to guarantee that this space will exist?

Something like
```python
@pytest.fixture(scope="session")
def diagram_space(cognite_client: CogniteClient) -> Space:
    return cognite_client.data_modeling.spaces.apply(SpaceApply(space=DIAGRAM_SPACE))
```

Reply:
Hm, could do that, but should also make sure that the file exists in that case. If the space does not exist, then the file cant exist either, so maybe add a file to the repo and upload it if it is not there or what do you think?

Reply:
Yes, I missed that. I would also make another fixture for the file then. If the file is not too big or sensitive then commit it to the repo.